pages/linux/hlint.md

```diff
@@ -0,0 +1,23 @@
+# hlint
+
+> Tool for suggesting improvements to Haskell code. These include alternative functions, simplifying code and spotting redundancies. (http://hackage.haskell.org/package/hlint).
```

Comment:
Let's remove the 2nd sentence to keep the line short. And move the link to a separate "More information: " line. Please see other pages for an example.